lib/jellyfish/room_service.ex

```diff
@@ -192,7 +191,38 @@ defmodule Jellyfish.RoomService do
   end
 
   defp get_resource_usage() do
-    list_rooms() |> Enum.count()
+    room_ids = get_rooms_ids()
+
+    Enum.each(room_ids, &Room.get_forwarded_tracks_in_room(&1))
+
+    accumulate_resource_responses(room_ids)
+    |> Enum.reduce(0, fn {_room_id, forwarded_tracks}, acc ->
+      acc + forwarded_tracks
+    end)
+  end
+
+  defp accumulate_resource_responses(room_ids, acc \\ []) do
+    receive do
+      response ->
+        acc = [response | acc]
+
+        if Enum.count(acc) == Enum.count(room_ids) do
+          acc
+        else
+          accumulate_resource_responses(room_ids, acc)
+        end
```

Comment:
If I was very pedantic (I'm not) I'd say you could add simply the number of forwarded tracks to the `acc` insteady of tuple `{room_id, forwarded_tracks}` and simply `Enum.sum/1` instad of the `Enum.reduce/3`.